binr/r2r/r2r.c

```diff
@@ -663,7 +663,7 @@ static void print_diff(const char *actual, const char *expected, bool diffchar,
 		d->diff_cmd = "git diff --no-index --word-diff=porcelain --word-diff-regex=.";
 	}
 	char *uni = r_diff_buffers_to_string (d, (const ut8 *)expected, (int)strlen (expected),
-	                                      (const ut8 *)output, (int)strlen (output));
+		(const ut8 *)output, (int)strlen (output));
```

Comment:
This was wrong, indenting up to the first parameter of the function is a bad idea, but it should be 2 tabs